### ISSUE TITLE
When parsing JSON response from Google, use parse_float option

### ISSUE THREAD

**Issue by commadelimited** [slimkrazy/python-google-places#34]
We're using Python 2.6.5 and when reviewing lat/lng values within our codebase they're incorrect, from what Google shows when [making a direct call](https://maps.googleapis.com/maps/api/place/details/json?placeid=ChIJSbQCipNmZIgRm4c6Nz9sGaE&key=<api-key>):

```python
# python 2.7.5
{
    lat: 36.151442,
    lng: -86.796573
}
```

versus
```python
# python 2.6.5
{u'lat': 36.151442000000003, u'lng': -86.796572999999995}
```
We found an option for parsing floats when loading JSON. This might be something worth implementing:

json.loads(x, parse_float = decimal.Decimal)

**Comment by commadelimited** [comment]
Hey @slimkrazy, any thoughts about making this change?

**Comment by slimkrazy** [comment]
Sorry for the delay on this! I'm just pushing this change to the development branch shortly. I'll be make a few more changes hopefully today, then will be releasing a new version with many fixes in a day or so.

**Comment by slimkrazy** [comment]
Released in version 1.1.0

**Comment by commadelimited** [comment]
Great. Thank you!
